docs/01-start-here/06-testing-tips.md

```diff
@@ -0,0 +1,157 @@
+- To test on a variety of devices or to reproduce a bug on a particular set up, use https://www.browserstack.com. This site has a collection of computers and devices,
+and many different versions of browsers. It is a very useful testing resource. Contact the dotcom team to get added to our user account for this tool
+
+### What to test on
+A list of useful information on device testing follows
+[Our browser support principals](https://github.com/guardian/frontend/blob/master/docs/04-quality/02-browser-support-principles.md);
+[A list of what browsers we support](https://github.com/guardian/frontend/blob/master/docs/04-quality/01-browser-support.md);
+[Ranking of the current top browsers](https://github.com/guardian/frontend/blob/master/docs/04-quality/01-browser-support.md)
+[Breakdown of last seven days web traffic by type of device](https://dashboard.ophan.co.uk/graph/breakdown?days=7&device=Personal+computer&device=Smartphone&device=Tablet&by=device)
+
+### Good places/tricks to find bugs
+- iPhone - Safari is the most popular browser on iOS (which makes it one of the most popular ways to view our site) and even Chrome on iOS has to use the iOS Webkit framework.
+ It is interesting from a testing perspective because it is a different operating system, different browser setup and different screen-size from the machine you are developing on.
+ This combination often throws up interesting layout problems. In addition Safari often has less support for new features compared with Chome, which can also lead to rendering errors.
+
+- IE9 - This is the second most popular IE version. IE is notorious for having less support for features and also being much less tolerant of small errors than Chrome, or Firefox. Often when a page renders on those browsers it will still have problems on IE
+ Add to this the fact that it is much older than IE11 so supports even less features. IE9 is the oldest browser version that we expect to be fully supported. If it works on IE9 it is a good indication it will work on most other browsers.
+
+- Chrome and Firefox on windows 10 - IE10 is slowly overtaking Windows7 as our users most popular desktop OS. When Windows 10 originally came out, firefox had some rendering issues and Chrome and firefox periodically show redering issues there - especially with things like fonts.
+
+- Older devices - A common source of problems comes from older devices. Where the device is running the latest operating system, it will attempt to run all the latest features and give the fullest experience. This can be catastropihc if the device just doesnt have the memory or processing power to cope.
+  This is especially important if your feature is very resource heavy. For reasons unknown, but possibly connected to the way safari manages memory, iOS devices have proved more susceptable
```

Comment:
Worth mentioning the special rules we have for iPad fronts due to this fact, would be useful to call out?